backend/routes/donation.js

```diff
@@ -88,16 +89,21 @@ router.post("/", (req, res) => {
                         .send(
                           "Error has occured during the post of the new donation !"
                         );
-                    return res
-                      .status(201)
-                      .send("Donation posted successfully !");
+                    res.status(201).send("Donation posted successfully !");
+                    sendMail({
+                      from: "Team.Meex@meex.com",
+                      to: email,
+                      replyTo: "Team.Meex@meex.com",
```

Comment:
(By convention, you should use 'no-reply@meex.com' if the address doesn't exist)